getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,73 @@
+# Instructions for macOS
+
+Install [Homebrew](http://brew.sh/) to give your mac a package manager. Open up your terminal emulator of choice and run:
+
+```shell
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
```

Comment:
Should we just include a link to [Homebrew](http://brew.sh)? That would help keep this doc simple and clear. Also, on the off chance that the install instructions change, we would still be giving correct information.